server.js

```diff
@@ -38,28 +36,117 @@ var renderHtml = function(url, cb) {
             }, 10000); //If no callPhantom script was called on the page, then wait 10secs and load page anyway.
         });
     };
+
+    console.debug("Opening page");
     page.open(url);
 
     page.onError = function (msg, trace) {
-        console.log(msg);
+        console.error(msg);
         trace.forEach(function(item) {
-            console.log('  ', item.file, ':', item.line);
+            console.error('  ', item.file, ':', item.line);
```

Comment:
well, it seems that Phantom doesn't yet allow for the formatting/filtering of the different kind of log messages. But in general this would allow for disabling debug logging and keeping the rest.

If phantom won't implement anything, than doing a `grep "console.debug"` and removing those lines is easier than to look trough all `console.log` lines and determining their importance.